update readme (#1)
DevExpress-Examples/winforms-filter-lookup-column-based-on-another-lookup-column: Readme.md

[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E898)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->

# Lookup Editors for Windows Forms - How to filter a second Lookup column based on a first LookUp column's value within a Grid Control

> In v16.1 and higher, a standalone Lookup editor supports automatic filtering of its popup data source based on a value of another standalone Lookup editor. See the following topic for more information: [Cascading Lookups](https://documentation.devexpress.com/WindowsForms/116018/Controls-and-Libraries/Editors-and-Simple-Controls/Lookup-Editors/Cascading-Lookups).

Reviewer: ```suggestion
In v16.1 and higher, a standalone Lookup editor allows you to automatically filter its popup data source based on the value of another standalone Lookup editor. See the following topic for more information: [Cascading Lookups](https://documentation.devexpress.com/WindowsForms/116018/Controls-and-Libraries/Editors-and-Simple-Controls/Lookup-Editors/Cascading-Lookups).
```
